<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelSessionBox</class>
 <widget class="QGroupBox" name="SelSessionBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>214</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Sessions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableView" name="sessionView"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="earlierBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>earlier</string>
        </property>
        <property name="icon">
         <iconset resource="resuurces/resources.qrc">
          <normaloff>:/arrow-down-double.png</normaloff>:/arrow-down-double.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="laterBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>later</string>
        </property>
        <property name="icon">
         <iconset resource="resuurces/resources.qrc">
          <normaloff>:/arrow-up-double.png</normaloff>:/arrow-up-double.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="filterBox">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="filterCheck">
          <property name="text">
           <string>&amp;filter</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+F</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="dateCheck">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="date">
           <date>
            <year>2012</year>
            <month>12</month>
            <day>12</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>dd/MM/yyyy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="userLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>User</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="userCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="serialLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>DDA serial</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="serialCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resuurces/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>sessionView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>SelSessionBox</receiver>
   <slot>onSessionChanged(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelSessionBox</receiver>
   <slot>onFilterChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>SelSessionBox</receiver>
   <slot>onDateChechChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>SelSessionBox</receiver>
   <slot>onDateChanged(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SelSessionBox</receiver>
   <slot>onUserChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serialCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SelSessionBox</receiver>
   <slot>onSerialChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSessionChanged(QModelIndex)</slot>
  <slot>onFilterChanged(bool)</slot>
  <slot>onDateChechChanged(bool)</slot>
  <slot>onDateChanged(QDate)</slot>
  <slot>onUserChanged(int)</slot>
  <slot>onSerialChanged(int)</slot>
 </slots>
</ui>
